ouroboros-consensus/src/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -216,11 +221,11 @@ instance (PBftCrypto c, Typeable c) => OuroborosTag (PBft c) where
             $ throwError (PBftExceededSignThreshold totalSigners gkSigners)
           return $! takeR (winSize + 2*k) chainState Seq.|> (gk, blockSlot b)
     where
-      PBftParams{..} = pbftParams
+      PBftParams{pbftSignatureThreshold=PBftSignatureThreshold sigThresh, ..} = pbftParams
       PBftFields{..} = headerPBftFields cfg b
       winSize = fromIntegral pbftSignatureWindow
       SecurityParam (fromIntegral -> k) = pbftSecurityParam
-      wt = floor $ pbftSignatureThreshold * fromIntegral winSize
+      wt = floor $ sigThresh * fromIntegral winSize
```

Comment:
Let's not mess with this in a PR about configuration plumbing. This is critical consensus code.

Reply:
Done!